packages/flutter/lib/src/material/flat_button.dart

```diff
@@ -29,7 +29,7 @@ import 'theme_data.dart';
 /// interactive, with ink splashes, without also committing to these stylistic
 /// choices, consider using [InkWell] instead.
 ///
-/// If the [onPressed] callback is null, then the button will be disabled,
+/// If [onPressed] or [onLongPress] callbacks are null, then the button will be disabled,
```

Comment:
If the [onPressed] and [onLongPress] callbacks are null, then this button will be disabled.